### DISCUSSION TITLE
Since yesteday I can only create files with 578 lines maximum

### DISCUSSION THREAD

**Discussion by Kees1958** [community/community#57832] [New to GitHub]
For some reason Github limits the number of lines I can upload only 578 lines (of a ABP format blocklist). The total number of rules are over 6500 lines. The file size of this text file is 10KB, so it is a small file.

Dragging the file, importing the file, creating a new file and copy-pasting all lines all result in only copying the first 578 lines

In he past I could simply copy-paste a text fiule to a github file.

What am I doing wrong?

**Comment by Rimkomatic** [comment]
I am not quite sure what is the issue, but I think you have accidently limited the amount of interactions you can do  go to [here](https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository) for more details

**Comment by airtower-luna** [comment]
I'm afraid I can't say *why* that's happening (could be a limitation of your clipboard, but that'd be odd to appear suddenly, or a bug in the GitHub web UI). What I'd suggest though is to actually use *Git* instead of fighting with the *GitHub* web UI. Clone the repository, add your file, commit, push. Make a PR if the file is meant for someone else's repository. :slightly_smiling_face: 

If you're not familiar with Git yet, I recommend you look at the Git Book, chapters 1-3 cover the basics: https://git-scm.com/book/en/v2
